src/main/resources/com/google/api/codegen/csharp/gapic_snippets.snip

```diff
@@ -0,0 +1,229 @@
+@extends "csharp/common.snip"
+
+@snippet generate(api)
```

Comment:
Done.
It actually does allow identical function and parameter names, but I've changed the function as you suggest for clarity.